### ISSUE TITLE
Dark mode colors have low contrast

### ISSUE THREAD

**Issue by Clashsoft** [twbs/bootstrap#38525]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The new darkmode colors make text very hard to read due to bad contrast. Compare these screenshots:

Unmodified:

![Unmodified](https://cdn.discordapp.com/attachments/626804176902684692/1101426600890867763/Screenshot_2023-04-28_at_10.35.35.png)

Text Color #eee:

![Text Color #eee](https://cdn.discordapp.com/attachments/626804176902684692/1101431842755854336/Screenshot_2023-04-28_at_10.56.42.png)

Text Color #eee and Body Color #111 (bootstrap-darkmode):

![Text Color #eee and Body Color #111 (bootstrap-darkmode)](https://cdn.discordapp.com/attachments/626804176902684692/1101431692482314250/Screenshot_2023-04-28_at_10.56.12.png)

I propose changing the `$body-color-dark` to something brighter like #eee for improved readability.

### Reduced test cases

-

### What operating system(s) are you seeing the problem on?

Windows, macOS, Android, iOS, Linux

### What browser(s) are you seeing the problem on?

Chrome, Safari, Firefox, Microsoft Edge, Opera

### What version of Bootstrap are you using?

v5.3.0-alpha3

**Comment by kusma** [comment]
FWIW, we've using the dark-mode (based on preference with JS) on [docs.mesa3d.org](https://docs.mesa3d.org), and we've gotten [complaints](https://gitlab.freedesktop.org/mesa/mesa/-/issues/9043) from users about the lack of contrast in the dark-mode as well.

**Comment by mdo** [comment]
The color contrast in dark mode (`#adb5bd` to `#1a1c20`) is 8.26. I could see us going a little darker on the background and perhaps brighter on the foreground.

**Comment by Clashsoft** [comment]
I see that the contrast works in theory, but as mentioned by @kusma and in my opinion, in practice it is still not readable for everyday text. Compare the comments here on GitHub, with a contrast ratio of 12.26:1.

**Comment by mdo** [comment]
Fixed by #38815.
